### PR TITLE
[NE-808]- RN for External DNS Operator

### PR DESCRIPTION
**Pull request by xenolinux**
Adds release notes for External DNS Operator Tech Preview feature
[Jira](https://issues.redhat.com/browse/NE-808)
[preview](https://deploy-preview-42710--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-networking-external-dns-operator)
QE contact- @lihongan PTAL

### CODE REVIEW

**Comment by lihongan** [comment]
LGTM. 
Thank you @xenolinux

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -539,6 +539,13 @@ Benefits of this feature include faster data processing, reduced CPU workloads,
 
 For more information, see xref:../networking/hardware_networks/configuring-hardware-offloading.adoc#about-hardware-offloading_hardware-offloading[Configuring hardware offloading].
 
+[id="ocp-4-10-networking-external-dns-operator"]
+==== Creating DNS records by using Red Hat External DNS Operator (Technology Preview)
+
+You can now create DNS records by using Red Hat External DNS Operator on cloud providers such as AWS, Azure and GCP. You can install the External DNS Operator using the Red Hat OpenShift Container Platform Operator Hub. You can use configurational parameters to configure `ExternalDNS` as per your need. 
+
+For more information, see xref:../networking/external_dns_operator/nw-installing-external-dns-operator.adoc#nw-installing-external-dns-operator[Installing External DNS Operator].
```
Comment (bergerhoffer) [inline release_notes/ocp-4-10-release-notes.adoc:547]: Installing External DNS Operator -> Installing the External DNS Operator

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -539,6 +539,13 @@ Benefits of this feature include faster data processing, reduced CPU workloads,
 
 For more information, see xref:../networking/hardware_networks/configuring-hardware-offloading.adoc#about-hardware-offloading_hardware-offloading[Configuring hardware offloading].
 
+[id="ocp-4-10-networking-external-dns-operator"]
+==== Creating DNS records by using Red Hat External DNS Operator (Technology Preview)
```
Comment (bergerhoffer) [inline release_notes/ocp-4-10-release-notes.adoc:543]: by using Red Hat External DNS Operator -> by using the Red Hat External DNS Operator

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -539,6 +539,13 @@ Benefits of this feature include faster data processing, reduced CPU workloads,
...
+
+You can now create DNS records by using Red Hat External DNS Operator on cloud providers such as AWS, Azure and GCP. You can install the External DNS Operator using the Red Hat OpenShift Container Platform Operator Hub. You can use configurational parameters to configure `ExternalDNS` as per your need.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-10-release-notes.adoc:545]: A few minor suggestions, just remember not to click the "commit suggestion" button, and instead make the updates manually.

```suggestion
You can now create DNS records by using the Red Hat External DNS Operator on cloud providers such as AWS, Azure, and GCP. You can install the External DNS Operator using OperatorHub. You can use parameters to configure `ExternalDNS` as required. 
```

**Review by bergerhoffer — COMMENTED** [review]
A few suggestions!

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (xenolinux) [reply release_notes/ocp-4-10-release-notes.adoc:547]: Addressed

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (xenolinux) [reply release_notes/ocp-4-10-release-notes.adoc]: Addressed

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
Comment (xenolinux) [reply release_notes/ocp-4-10-release-notes.adoc]: Addressed

**Review by bergerhoffer — COMMENTED** [review]
LGTM now, thanks!

**Pull request merged**